lang/en/accredible.php

```diff
@@ -75,5 +74,6 @@
 $string['viewsubheader'] = 'Group ID: {$a}';
 
 $string['gotodashboard'] = 'To update the appearance of your badges and certificates, visit: <a href="https://dashboard.accredible.com" target="_blank">https://dashboard.accredible.com</a>';
-$string['activitydescription'] = 'This module automatically creates and updates a Credential Group on Accredible. Any certificates and badges issued are in this group and their appearance and information can be updated on <a href="https://dashboard.accredible.com" target="_blank">the dashboard</a>.';
 $string['overview'] = 'Overview';
+$string['activitygroupdescription'] = 'Credentials groups need to have been created in the <a href="{$a}" target="_blank">Accredible Dashboard</a> before credentials can be issued. If none appear, check your API Key to make sure integration is set up correctly.';
```

Comment:
Question - Have you checked this wording with the product team?

Reply:
No I haven't. I added the same wording we have in Blackboard with the instructions about the Accredible Groups. Who is the best person to ask about it?

Reply:
In that case, I think it should be fine, but let's make sure of it with Alex before merging this into `develop`.